my_list.rb

```diff
@@ -0,0 +1,19 @@
+require './my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize
+    @list = []
+  end
```

Comment:
- [ ] when I tried to verify your app with an example in the requirements it doesn't work cause you didn't add a way to accept multiple arguments so please make sure to add that in order to verify you app is working 
```ruby 
# Create our list
list = MyList.new(1, 2, 3, 4)

# Test #all?
p list.all? {|e| e < 5}
p list.all? {|e| e > 5}

# Test #any?
p list.any? {|e| e == 2}

p list.any? {|e| e == 5}

# Test #filter
p list.filter {|e| e.even?}

=> #<MyList: @list=[1, 2, 3, 4]>
```